setup.cfg

```diff
@@ -0,0 +1,18 @@
+[flake8]
+ignore = W503, E203, B305
+max-line-length = 88
+
+[mypy]
+disallow_untyped_defs = True
+ignore_missing_imports = True
+
+[mypy-tests.*]
+disallow_untyped_defs = False
+# check_untyped_defs = True
```

Comment:
This doesn't pass yet. Not sure whether to remove or add a TODO comment or something?

Reply:
Probably remove this `tests.*` section and log a ticket to "Add type checking to test suite"?

Reply:
I created #1045 (open issue number 100 😰). I think it makes sense to leave `mypy-tests.*` in here and have mypy run against the tests directory like we do so that it can still check (very basic) things and at least we don't regress.